### ISSUE TITLE
Debug platform and build plan

### ISSUE THREAD

**Issue by msteinhoff** [holos-run/holos#331]
I can render the platform via `holos render platform ./platform`, but this is all or nothing.

When something goes wrong, I would like to inspect intermediate stages to be able to pinpoint misconfigurations.

In the tutorial I found `cue export --out yaml ./platform` to render the platform with all components, and `cue export --expression holos --out=yaml ./components/podinfo` to render an individual component, but those feel clunky to use.

It would be nice if holos could provide a command to print both the platform definition and a build plan for all components to stdout without actually rendering them.

**Comment by jeffmccune** [comment]
Agreed, something like `holos show platform ./platform` and `holos show component ./components/podinfo` are commands I'd like to add sooner rather than later.

FYI, right now, if you run in full debug mode you can get the tags injected into the component which gives you the complete context necessary to export the build plan with CUE.

For example:

```
holos render platform ./platform --log-level=debug --log-format=json 2> /tmp/log.json
```

```
jq -r 'select(.args != null) | .args | join(" ")' /tmp/log.json
```

This will give you:

```
render component --inject holos_component_name=podinfo --inject holos_component_path=components/podinfo components/podinfo
```

Clunky for sure, you can copy and paste this remembering to add `./` to the component path:

```
holos cue export --out=yaml --inject holos_component_name=podinfo --inject holos_component_path=components/podinfo ./components/podinfo
```

**Comment by jeffmccune** [comment]
Is it a requirement to get all `BuildPlan` resources in one YAML stream?  What's the use case for this, so you can slice and dice it with `yq`?

**Comment by jeffmccune** [comment]
@msteinhoff FYI I added `holos show buildplans` and `holos show platform` commands in #349 released in `v0.100.0`.

```
❯ holos show buildplans --help
Show BuildPlans produced from Platform.spec.components

1. Selectors are applied to the Platform.spec.components list.
2. Results are output in the same order as listed in the Platform spec.

Usage:
  holos show buildplans [flags]

Aliases:
  buildplans, buildplan, components, component

Flags:
      --platform string     platform directory path (default "./platform")
      --format string       yaml or json format (default "yaml")
  -l, --selector selector   label selector (e.g. label==string,label!=string)
  -t, --inject tags         set the value of a cue @tag field from a key=value pair
      --concurrency int     number of concurrent build steps (default 8)
  -v, --version             version for buildplans

Global Flags:
      --log-drop strings    log attributes to drop (example "user-agent,version")
      --log-format string   log format (text|json|console) (default "console")
      --log-level string    log level (debug|info|warn|error) (default "info")
```
